netweaver/monitoring.sls

```diff
@@ -0,0 +1,33 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+
+{% for node in netweaver.nodes if host == node.host %}
```

Comment:
Even though it doesn't affect, I will use the `for` loop only in the configuration file creation. The other 2 states can be executed outside the loop without any problem